dataverse-webapp/src/main/webapp/resources/vecler/theme.scss

```diff
@@ -279,12 +264,83 @@ input, textarea {
 	}
 }
 
-.field-add-delete {
-	margin-left: -1.5em;
+/* Forms */
+
+.ui-selectonemenu {
+	border: 1px solid #e8e8e8;
+
+	.ui-selectonemenu-label {
+		padding-left: 15px;
+		background: #fff;
+		box-shadow: none;
+	}
+
+	.ui-selectonemenu-trigger.ui-state-default {
+		width: 44px;
+		top: -1px;
+		right: -1px;
+		padding-bottom: 32px;
+		border: 1px solid $gray-border-color;
+		border-radius: 4px;
+		background: #fff;
+
+		&:hover {
+			background: $gray-border-color;
+		}
+
+		span.ui-icon.ui-icon-triangle-1-s {
+			width: 42px;
+			margin-top: 7px;
+			background: none;
+			text-align: center;
+
+			&::before {
+				content: "\e252";
+				position: absolute;
+				left: 0;
+				right: 0;
+				font-family: "Glyphicons Halflings";
+				font-size: 0.8em;
+				color: $icon-color;
+				text-indent: 0;
+			}
+		}
+	}
+}
+
+div.field-add-delete {
+	margin-left: -1.2em;
+
+	a.btn {
+		margin-right: 0.6em;
+	}
+
 
 	.btn span.glyphicon.no-text {
 		margin: 0 1em;
 	}
+
+	@media (max-width: 991.98px) {
```

Comment:
It looks like we are looking for a breakpoint compatible with bootstrap here.
I would declare this as a variable with naming that will point that it is for maximum range of small screen sizes
In bootstrap itself it is done with:
```
@screen-md:                  992px;
@screen-md-min:              @screen-md;
@screen-sm-max:              (@screen-md-min - 1);

```
https://github.com/twbs/bootstrap/blob/v3.4.1/less/variables.less
(probably if we use pixels then specifying value as float is not needed 991px is ok)

Reply:
Fixed